src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -233,7 +233,12 @@ object TezosTypes {
   final case object DoubleEndorsementEvidence extends Operation
   final case object DoubleBakingEvidence extends Operation
   final case object Proposals extends Operation
-  final case object Ballot extends Operation
+  final case class Ballot(ballot: Vote) extends Operation
```

Comment:
You might need to look into decodings and conversions tests both to add tests for the new operation detail